jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -128,6 +132,32 @@ public Server(@Name("threadpool") ThreadPool pool)
         addBean(_threadPool);
         setServer(this);
         addBean(FileSystemPool.INSTANCE, false);
+        setDefaultHandler(new DefaultHandler());
+    }
+
+    public Handler getDefaultHandler()
+    {
+        return _defaultHandler;
+    }
+
+    /**
+     * @param defaultHandler The handler to use if no other handler is set or accepts the request. This handler should
+     *                       always accept the request, even if only to send a 404.
+     */
+    public void setDefaultHandler(Handler defaultHandler)
+    {
+        if (isStarted())
```

Comment:
If `isDynamic()` allow to set a `DefaultHandler` ?

Reply:
maybe... but that is getting a bit meta.  Let me think on that one.

Reply:
ok, added isDynamic check